src/heros/solver/FlowFunctionDotExport.java

```diff
@@ -36,6 +37,27 @@
  * @param <I> The type of inter-procedural control-flow graph being used.
  */
 public class FlowFunctionDotExport<N,D,M,I extends InterproceduralCFG<N, M>> {
+	private static class Numberer<D> {
```

Comment:
As Heros is independent of Soot, I just hacked up a quick replacement for Numberer (specifically MapNumberer). If there is a existing utility in Heros I should use instead I would prefer that (although there is precedent for this in the Pair type?)